client/src/app/App.js

```diff
@@ -2007,11 +2009,18 @@ export class App extends PureComponent {
     });
 
     const groupedItems = map(groupBy(items, 'group'), (group, key) => {
-      return {
+
+      var item = {
         items: group,
         key,
-        label: key
+        label: key,
       };
+
+      if (key === 'Camunda Platform 8') {
+        item.labelSuffix = <NewBadge inline style={ { marginLeft: '20px', transform: 'translate(0px, -1px)' } } />;
+      }
```

Comment:
What about having a simple test case for `_getNewFileItems` whether it returns a label suffix for C8? Or is it so temporarily that we don't need it?

Reply:
No I agree with you I'll do that.